app/controllers/campaigns_controller.rb

```diff
@@ -44,12 +44,12 @@ def create
   end
 
   def edit
-    @repo_name = @campaign.repo_identifier
+    @repo_name = @campaign.repository
```

Comment:
```suggestion
    @repository = @campaign.repository
```

Anyway this is redundant as the `set_campaign` method already does that currently.

Reply:
👍